forem/forem
# Updated indicators

## What type of PR is this? (check all applicable)


## Description

This PR slightly updates our indicator (little colored badges with number or short label) as well as introduces proper Storybook documentation and controls.

It slightly updates the code:
- I renamed `crayons-indicator` to be `c-indicator`
- I added few variants (styles) and extra configuration (`relaxed` prop) needed for other projects.

This is something we wanted to update for Admin's Member Details view that is currently being in progress so I thought it's a good moment to update this.

We don't have too many instances of this component used across the codebase so I just went ahead and updated these everywhere, so `crayons-indicator` should no longer be there.

Keep in mind most of the changed files in this PR is simply coming from renaming `crayons-indicator` to `c-indicator` 🤷 

## Documents

[Figma source file](https://www.figma.com/file/gHpwugeu9jFGW4d9u3ReuR/Crayons?node-id=391%3A234) - keep in mind we have different colors in Figma because our Figma document already uses new colors palette which is not yet implemented in the codebase - it's currently being reviewed in [separate PR](https://github.com/forem/forem/pull/16107). I will of course update everything once we ship that PR.

## QA Instructions, Screenshots, Recordings

**Storybook:** Run `yarn storybook` and go to Indicators section which should now contain correct controls and documentation.
**Product:** Just browse around. Even though all indicators have been "updated" - changes aren't really visually significant or meaningful to raise anyone's concerns or to accidentally break anything.

### UI accessibility concerns?

I've checked the contrast ratio for each variant (style) for both themes (light & dark).

## Added/updated tests?

- [x] No, should I?

## [Forem core team only] How will this change be communicated?

Forem.team Crayons changelog post.
